configs/skl_with_context_config.json

```diff
@@ -0,0 +1,77 @@
+{
+    "common": {
+        "lib": ["sklearn"],
+        "data-format": ["pandas"],
+        "data-order": ["F"],
+        "dtype": ["float64"],
+        "device": ["host", "cpu", "gpu"]
```

Comment:
What happens if I run this config on a machine without a GPU driver?

Reply:
Exactly the same what happens if you try to run on a CPU wo DPC++ support - an exception. What is your suggession here?

Reply:
We shall point somewhere that using this config file requires DPC++ support and GPU device on board

Reply:
Ok, probably it's ok